app/controllers/admin/tenants_controller.rb

```diff
@@ -32,4 +33,20 @@ def update
     def tenant_params
       params.require(:tenant).permit(:name, :schema, :schema_type)
     end
+
+    def create_tenant_administrator
+      Tenant.switch(@tenant.schema) do
```

Comment:
Not necessarily bad, although this is the only time where we use `Tenant.switch` inside a request. It scares me a bit but I guess we're implicitly doing the same thing when we run `Apartment::Tenant.create` :relieved:.

So it should be OK, but let's keep an eye on it if we detect unusual behaviors in the future (like some tests failing sometimesbecause they access the wrong tenant).